README.md

```diff
@@ -1,4 +1,28 @@
-# Gateway API Inference Extension 
+# Gateway API Inference Extension (GIE)
+
+This project offers tools for [AI Inference], enabling developers to build [Inference Gateways].
+
+[AI Inference]:https://www.digitalocean.com/community/tutorials/llm-inference-optimization
+[Inference Gateways]:#concepts-and-definitions
+
+## Concepts and Definitions
+
+AI/ML is changing rapidly, and [Inference] goes beyond basic networking to include complex traffic routing and optimizations. Below are key terms used within this project:
+
+- **Scheduler**: Makes decisions about which endpoint is optimal (best cost / best performance) for an inference request based on `Metrics and Capabilities` from [Model Serving Platforms].
+- **Metrics and Capabilities**: Data provided by model serving platforms about performance, availability and capabilities to optimize routing. Includes things like [Prefix Cache] status or [LoRA Adapters] availability.
+- **Endpoint Picker**: A `Scheduler` combined with `Metrics and Capabilities` systems is often referred to together as an `Endpoint Picker (EPP)`.
+- **Inference Gateway**: A proxy/load-balancer which has been coupled with a `Endpoint Picker` to ensure optimized routing decisions for inference requests.
+
+For deeper insights and more advanced concepts, refer to our [proposals](/docs/proposals).
+
+[Inference]:https://www.digitalocean.com/community/tutorials/llm-inference-optimization
+[Gateway API]:https://github.com/kubernetes-sigs/gateway-api
+[Model Serving Platforms]:https://ubuntu.com/blog/guide-to-ml-model-serving
```

Comment:
I'm not a fan of this blog post as it highlights other projects and paid training.

Reply:
```suggestion
```